test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -305,11 +310,38 @@ func BenchmarkPerfScheduling(b *testing.B) {
 	}
 }
 
+func loadConfigFromFile(file string) (*config.KubeSchedulerConfiguration, error) {
```

Comment:
I assume this is from https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-scheduler/app/options/configfile.go - should we make the function public and re-use it instead?

Reply:
true, some refactoring will be done to remove the redundancy.

Reply:
I am not if expose the method directly will potentially  leads to a recursive or not in the future, so far it is safe.